WcaOnRails/app/views/polls/index.html.erb

```diff
@@ -0,0 +1,69 @@
+<% provide(:title, 'Polls') %>
+
+<div class="container">
+  <h1><%= yield(:title) %></h1>
+
+  <% can_create = current_user.can_create_poll? %>
+  <% if can_create %>
+    <%= link_to "New poll", new_poll_path, class: "new-poll" %>
+  <% end %>
+
+  <% if @polls.empty? %>
+    <div class="alert alert-warning">There are currently no polls.</div>
+  <% else %>
+    <table class="table polls-list">
+      <%# @polls.each do |poll| %>
+      <h4>Open polls</h4>
+      <% @openPolls.each do |poll| %>
+        <% if can_create && !poll.confirmed %>
```

Comment:
What's going on here? A poll is confirmed whether or not the user can create polls...